src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -27,6 +27,7 @@ const partyMapVenueSelector = (state: RootState) =>
   state.firestore.ordered.currentVenue?.[0] as PartyMapVenue;
 
 export const PartyMap: React.FC = () => {
+  useConnectCurrentVenue();
```

Comment:
[OOS] This should use `useConnectCurrentVenueNG`.

Reply:
It should be the rest of the code is using the default one and I don't have the time to change it atm.